Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -344,33 +344,35 @@ - (void)registerToken:(NSData *)token appName:(NSString *)appName pushServicePla
     [self sendRequest:request onSuccess:onSuccess onFailure:onFailure];
 }
 
-- (void)disableToken:(NSData *)token appName:(NSString *)appName pushServicePlatform:(PushServicePlatform)pushServicePlatform {
+// documented in IterableAPI.h
+- (void)disableToken:(NSData *)token
+{
+    return [self disableToken:token andEmail:nil onSuccess:[IterableAPI defaultOnSuccess:@"registerToken"] onFailure:[IterableAPI defaultOnFailure:@"registerToken"]];
+}
+
+// documented in IterableAPI.h
+- (void)disableToken:(NSData *)token andEmail:(NSString *)email
+{
+    return [self disableToken:token andEmail:email onSuccess:[IterableAPI defaultOnSuccess:@"registerToken"] onFailure:[IterableAPI defaultOnFailure:@"registerToken"]];
+}
+
+// documented in IterableAPI.h
+- (void)disableToken:(NSData *)token onSuccess:(OnSuccessHandler)onSuccess onFailure:(OnFailureHandler)onFailure
+{
+    return [self disableToken:token andEmail:nil onSuccess:onSuccess onFailure:onFailure];
+}
+
+// documented in IterableAPI.h
+- (void)disableToken:(NSData *)token andEmail:(nullable NSString *)email onSuccess:(OnSuccessHandler)onSuccess onFailure:(OnFailureHandler)onFailure
```

Comment:
I believe you don't need `nullable` here since it's already declared that way in the header